neutrino.go

```diff
@@ -985,6 +1030,17 @@ func NewChainService(cfg Config) (*ChainService, error) {
 	return &s, nil
 }
 
+// NewHTTPClient of type HTTPClient.
+func NewHTTPClient() (*http.Client, error) {
+	// We'll retun a reguar HTTP client.
+	return &http.Client{Timeout: 10 * time.Second}, nil
+}
+
+// Function queries the HTTPClient and with the requested url.
```

Comment:
Why a function instead of one line of code?

Reply:
Added a oneliner to query.go instead